runner/src/main/java/org/ananas/runner/steprunner/jdbc/pgsql/PostgresqlDataTypes.java

```diff
@@ -9,16 +9,24 @@
 
 public enum PostgresqlDataTypes implements JDBCDataType, DDL {
   BIGINT("bigint", Schema.FieldType.INT64.withNullable(true), true),
+  OID("oid", Schema.FieldType.STRING.withNullable(true), true),
+  XID("xid", Schema.FieldType.STRING.withNullable(true), true),
```

Comment:
Need to change the default to false, otherwise, all STRING type will map to "oid" by default